libs/application/templates/children-residence-change/src/fields/Overview/index.tsx

```diff
@@ -19,6 +21,28 @@ const Overview = ({ application }: FieldBaseProps) => {
   const children = extractChildrenFromApplication(application)
   const answers = extractAnswersFromApplication(application)
   const { formatMessage } = useIntl()
+
+  const [createPdfPresignedUrl, { loading: loadingUrl, data }] = useMutation(
+    CREATE_PDF_PRESIGNED_URL,
+    {
+      onError: (e) => console.log('error', e),
```

Comment:
Do we want to add a TODO comment here or do you feel that a ticket on our board is enough?

Reply:
I've added a ticket to our board. This will be a part of a bigger error handling ticket.